modules/modify-data-object/pkg/dataobject/provider.go

```diff
@@ -106,9 +116,14 @@ func NewDataObjectCreator(cliOptions *parse.CLIOptions) (*DataObjectCreator, err
 		return nil, err
 	}
 
+	k8sClient, err := k8sv1.NewForConfig(config)
```

Comment:
Might make sense to use this way of creating the cdiclient too, returning an error instead of panicking?